README.md

```diff
@@ -48,6 +48,20 @@ There are currently three main scripts available:
 
 For all of these, it's assumed that you are running from the `scripts/` subdirectory.
 
+### Running Tests: via Docker 
+
+You can easily run the tests included in this repo. These tests should be run, and pass, before submitting an update to the repo. 
+The tests are packaged in a `docker-compose` file for easy execution. Docker and Docker Compose must be installed to run tests using this method. 
+See the links at the end of this section for installation links.
+
+To run the tests, from the repo root, issue this command:
+`docker-compose -f docker-compose-test.yml run test`
+
```

Comment:
You should mention that there will be a file created in the local directory, due to the ./:/output mapping and the test command used in the docker-compose-test.yml